modules/ext.thanks.corethank.js

```diff
@@ -52,7 +52,11 @@
 			.then(
 				// Success
 				function () {
-					$thankElement.before( mw.message( 'thanks-thanked', mw.user, $thankLink.data( 'recipient-gender' ) ).escaped() );
+					// FANDOM change. Gave the "before" element a class, so we can style it.
+					$thankElement.before( `<span class="mw-thanks-thank-confirmation">
```

Comment:
I've managed to do that in a hacky way in PHP to avoid upstream modifications:
https://github.com/Wikia/unified-platform/pull/13734/files#diff-cae9af1d99d1aa22ebf74344705b2c6a0c954f71850875d130060ca18e5af526R51

Is it viable for this scenario?

Reply:
it's not. https://fandom.atlassian.net/browse/UGC-4107